src/hapi/pipelines/database/org.py

```diff
@@ -50,19 +54,38 @@ def populate_single(
         org_name,
         org_type,
     ):
-        org_row = DBOrg(
-            acronym=acronym,
-            name=org_name,
-            org_type_code=org_type,
+        key = (
+            clean_name(acronym).upper(),
+            clean_name(org_name).upper(),
         )
-        self._session.add(org_row)
-        self._session.commit()
```

Comment:
This function should be renamed since it doesn't add the rows to the db table and populate is used elsewhere to refer to adding rows to tables. Not sure what the best naming is maybe add_org?

Reply:
Yes I'm also left a bit confused about what all the org methods are doing exactly, a rename of some of them would help me out.